test_cases/stdlib/builtins/test_dict.py

```diff
@@ -0,0 +1,37 @@
+# pyright: reportUnnecessaryTypeIgnoreComment=true
+
+from typing import Dict, Generic, Iterable, NoReturn, Tuple, TypeVar
+from typing_extensions import assert_type
+
+# These do follow `__init__` overloads order:
+assert_type(dict(), Dict[NoReturn, NoReturn])
```

Comment:
This one fails pyright, and also doesn't seem very idiomatic. You'd never deliberately annotate something as `dict[NoReturn, NoReturn]` in real code. Something like this should pass mypy and pyright, and be more idiomatic:

```suggestion
bad = dict()  # type: ignore
good: Dict[str, str] = dict()
assert_type(good, Dict[str, str])
```

Reply:
Do you know why I can't press "add suggestion to batch" in this repo? 🤔

Reply:
Is this feature disabled in the settings or something?